setup.py

```diff
@@ -36,6 +36,6 @@
         ]
     },
     extras_require={
-        'vectors': ['numpy', 'scipy', 'statsmodels', 'tables', 'pandas', 'scikit-learn', 'mecab-python3', 'jieba']
+        'vectors': ['numpy', 'scipy', 'statsmodels', 'tables', 'pandas', 'scikit-learn', 'mecab-python3', 'jieba', 'marisa_trie']
     },
 )
```

Comment:
I've been dealing with the added dependency on wordfreq with jieba and mecab. marisa_trie should be an easy dependency, though, that just gets installed via setup.py.